spec/api-server.spec.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+describe("ApiServer", () => {
+    var apiServer, webServerMock;
+
+    beforeEach(() => {
+        var ApiServer, configMock;
+
+        ApiServer = require("../lib/api-server");
+        configMock = {
```

Comment:
I would simplify this test by eliminating configMock from here entirely and moving server creation to within the test.  Then you can say things like `var config = {}` and `expect(webServerMock).toHaveBeenCalledWith(config);`.